Projects/05matchGame/index.js

```diff
@@ -0,0 +1,402 @@
+(async function initFunc() {
+  let isNotMobile = window.matchMedia(
```

Comment:
1. Rename the variable to "isMobile".
2. Find better way to detect resolution or mobile device, read this question - https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser